pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-matrix/default.nix

```
@@ -4,6 +4,8 @@ stdenv.mkDerivation rec {
   pname = "purple-matrix-unstable";
   version = "2019-06-06";
 
+  buildFlags = "CC=cc";
```

```suggestion
  buildFlags = [ "CC=cc" ];
```
It does not matter right now but it well when structuredAttrs land.